pkg/build/api/types.go

```diff
@@ -512,7 +512,7 @@ type SourceBuildStrategy struct {
 	Scripts string
 
 	// Incremental flag forces the Source build to do incremental builds if true.
-	Incremental bool
+	Incremental *bool
```

Comment:
In fact, given `omitempty`, it's impossible for me to have created a buildconfig before that will have this forced to `false`.  I'll always be at the mercy of the default.